### PR TITLE
3rd party notification system issue #207

### PR DESCRIPTION
**Pull request by Hrishikesh-Bhagwat**
**What kind of change does this PR introduce?**

This PR adds the feature for push notifications using 3rd party and solves issue #207

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
No
**Summary**
I feel push notifications are necessary for an application to attract more users and keep them engaged.

**Does this PR introduce a breaking change?**
No

**Other information**
We used a 3rd party FCM plugin to send push notifications.

### CODE REVIEW

**Review by XshubhamX — COMMENTED** [review]
Hey just a suggestion ,try to divide your pr in 2 - 3 commits,it makes easy to review code and maintain stability. btw nice work @Hrishikesh-Bhagwat

**Comment by palisadoes** [comment]
@DMills27 @Laurell876 please review

**Review by Laurell876 — APPROVED** [review]
LGTM

File: resolvers/push_notifications/notifications.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        res.json({
+          message: "Notification sent!",
+          success: true
+      })
+      })
+      .catch((error) => {
+        console.log(error);
+        res.json({
+          message: "Notification not sent:(",
+          success: true
```
Comment (impiyush83) [inline resolvers/push_notifications/notifications.js:87]: @Hrishikesh-Bhagwat  This should be false

Comment (impiyush83) [reply resolvers/push_notifications/notifications.js:87]: @Laurell876  @palisadoes

File: resolvers/push_notifications/notifications.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          })
+          });
+});
+
+router.post('/notify-topic', async (req, res) => {
+    var topic=req.body.topic;
+    var messageTitle = req.body.title;
+    var messageBody = req.body.message;
+      
+      const message = {
```
Comment (impiyush83) [inline resolvers/push_notifications/notifications.js:68]: @Hrishikesh-Bhagwat  Always use const as you are not re-instantiating the variables. 

Use restructuring,

const { title: messageTitle, message: messageBody, topic } = req.body;

@palisadoes  @Laurell876  @Sagar2366

File: resolvers/push_notifications/notifications.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+const express = require("express");
+const router = express.Router();
+const admin = require("firebase-admin");
+var serviceAccount = require('PATH TO SERVICE ACCOUNT JSON KEYS');
```
Comment (impiyush83) [inline resolvers/push_notifications/notifications.js:4]: use const instead of var

**Pull request merged**